dashboard/constants.bal

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// Names
+const BALLERINA_ORG_NAME = "ballerina-platform";
+const BALLERINA_STANDARD_LIBRARY = "ballerina-standard-library";
+
+// Links
+const BALLERINA_ORG_URL = "https://github.com/ballerina-platform";
+const GITHUB_BADGE_URL = "https://img.shields.io/github";
+const CODECOV_BADGE_URL = "https://codecov.io/gh";
+const GITHUB_API_LINK = "https://api.github.com/repos";
+const GITHUB_RAW_LINK = "https://raw.githubusercontent.com";
+
+// Colors
+const BADGE_COLOR_GREEN = "30c955";
+const BADGE_COLOR_YELLOW = "yellow";
+
+// File Paths
+const MODULE_LIST_JSON = "../release/resources/module_list.json";
+const STDLIB_MODULES_JSON = "../release/resources/stdlib_modules.json";
+const README_FILE = "../README.md";
+const GRADLE_PROPERTIES = "gradle.properties";
+
+// Retry Configs
+const HTTP_REQUEST_RETRIES = 3;
+const HTTP_REQUEST_DELAY_IN_SECONDS = 2;
+const HTTP_REQUEST_DELAY_MULTIPLIER = 2;
```

Comment:
You can provide the type when defining a constant. This way, you don't have to cast them later.

```suggestion
const decimal HTTP_REQUEST_DELAY_IN_SECONDS = 2;
const float HTTP_REQUEST_DELAY_MULTIPLIER = 2;
```